.github/workflows/pr_bench.yml

```diff
@@ -0,0 +1,25 @@
+name: Benchmark pull requests
+on: [pull_request]
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  runBenchmark:
+    name: run benchmark
+    runs-on: [self-hosted, bench]
```

Comment:
did you set up a specific bench runner?

Reply:
Ah, you did. Ideally, we'd have a way to make this work in 15min, even on hosted runners.